### DISCUSSION TITLE
Expand api rule

### DISCUSSION THREAD

**Discussion by Oggy4th** [pocketbase/pocketbase#2196] [Q&A]
Is it possible to expand relation field in api rule and check against string?

**Comment by ganigeorgiev** [comment]
You can use dot-notation to access relation fields, if that's what you are asking. For example:

```js
someRelField.title = "example"
```

**Reply by Oggy4th** [reply]
Hello, there
Thanks for your reply

Does it work?

![image](https://user-images.githubusercontent.com/44497354/229295154-cec11727-dadf-44d0-8269-6183e8970cba.png)
![image](https://user-images.githubusercontent.com/44497354/229295356-849e0066-5653-4a1b-81b3-f0303e04a9e6.png)

`employeeTypeId.Permissions` this field is multiple select field.

**Reply by ganigeorgiev** [reply]
Yes, the above should work if:
- `employeeTypeId` is a field in your auth collection
- the current authenticated record has at least one `employeeTypeId.permissions` value matching the text "GET".
